### ISSUE TITLE
A specific parameter for additional remote writes

### ISSUE THREAD

**Issue by abjrcode** [SumoLogic/sumologic-kubernetes-collection#1480]
Why is it not possible to add additional `remoteWrites` just like it is the case for [`additionalServiceMonitors`](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/deploy/helm/sumologic/values.yaml#L1278)?

The documentation on scraping additional services now points to the prometheus-overrides.yaml file, but that is error prone as I need to _(possibly)_ re-generate it on chart updates?

Basically it would be nice to add additional monitors and remote writes without touching the ones provided by Sumo _(just augmenting them)_.

**Comment by abjrcode** [comment]
Something to consider, I and probably most others implement this with GitOps and want to avoid running any commands manually if possible.

**Comment by perk-sumo** [comment]
Hi, the reason for that is because the upstream chart we use doesn't provide such possibility - there is only one section for remoteWrite to be used:
https://github.com/prometheus-community/helm-charts/blob/kube-prometheus-stack-12.3.0/charts/kube-prometheus-stack/values.yaml#L1943

We know it's not ideal.

**Comment by abjrcode** [comment]
Thanks for the clarification. Wouldn't be possible that you expose this as a parameter on your own chart and then merge the result when passing it to the subchart?

**Comment by sumo-drosiek** [comment]
> Wouldn't be possible that you expose this as a parameter on your own chart and then merge the result when passing it to the subchart?

We are not aware of possibility to do that and keep everything clean and simple

We created [PR](https://github.com/prometheus-community/helm-charts/pull/747) in the kube-prometheus-stack helm chart, so maybe we will be able to do it in a proper way :crossed_fingers:

**Comment by sumo-drosiek** [comment]
labeling as `breaking-change`, because it requires updating the Prometheus CRDs
